README.md

```diff
@@ -83,3 +99,14 @@ Type `"` in any `pipeline.py` and it should trigger the autocompletion list.
 ## Hover
 Just hover your mouse over any `params:`, datasets or hit the command `Show or Focus Hover`
 ![hover](assets/lsp-hover.gif)
+
+## To navigate between the flowchart and the code editor in Kedro-Viz:
+
+Navigate to Node Functions:
+Click on a node in the Kedro-Viz flowchart, and it will automatically navigate to the corresponding node function in your code.
```

Comment:
I'm pretty late to this, but I'm wondering: since the navigation functionality on the editor is `Cmd+Click` (see "Go to Definition from pipeline.py to configuration files" and "Go to Reference from configuration files to pipeline.py" above in this same README), and that clicking nodes has a different behavior on the Viz web application...

...should the navigation action for Viz on VS Code be `Cmd/Ctrl+Click` as well, for consistency?

cc @stephkaiser @iamelijahko

Reply:
Not sure if I like the Cmd+click, unless hitting cmd show something different (like in editor it will turn text into underlined blue text). Either way it's inconsisent either with the IDE or vis web.

Reply:
if I've understood correctly.. I think its okay to keep the 'clicking nodes' interaction on Viz because thats what users are familiar with and its consistent with how users currently interact with Viz. And I assume the Cmd+Click behaviour is also standard on VS code. I'd suggest to keep it like this, it makes the most sense to me for now.